contributingGuides/CONTRIBUTING.md

```diff
@@ -59,7 +59,7 @@ Please be aware that compensation for any support in solving an issue is provide
 - Merged PR within 9 business days - 50% **penalty**  
 - No PR within 12 business days - **Contract terminated**  
 
-If the PR causes a regression within 7 days of being deployed to production, contributors are not eligible for the 50% bonus. 
+If the PR causes a regression at any point before the end of the regression period (7 days after being deployed to production), contributors are not eligible for the 50% bonus. 
```

Comment:
Being deployed happens after merging, so this wouldn't be correct. @mallenexpensify I wonder though if we should call out that Melvin will autopost informing of the deployment, so that everyone's on the same page about when that happens?

Reply:
Thanks @getusha and @dangrous , we went back and forth (and back and forth) on the wording and I still wasn't certain the final version was the most clear.  @dangrous can you propose updated language you think would work better?